### DISCUSSION TITLE
Getting + icon on fields after upgrade to 10.0.7

### DISCUSSION THREAD

**Discussion by vinitchopra12** [pimcore/pimcore#9719] [Q&A / Support] [answered]
Hello,

I am getting + icon in all fields after I did the upgrade from pimcore 10.0.6 to 10.0.7.

![image](https://user-images.githubusercontent.com/14720025/125162413-49dce980-e1a5-11eb-896a-0164f3211d08.png)

**Comment by fashxp** [comment, marked as answer]
It's not a bug, it's a feature. You can turn this off in the system settings. 
Please see https://pimcore.com/docs/pimcore/current/Development_Documentation/Multi_Language_i18n/Admin_Translations.html and https://github.com/pimcore/pimcore/pull/9678
